go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/GoogleContainerTools/kpt
 go 1.17
 
 require (
-	github.com/GoogleContainerTools/kpt/porch/api v0.0.0-20220404223850-4fe7ba260b65
+	github.com/GoogleContainerTools/kpt/porch/api v0.0.0-20220407000100-7a5545349d71
```

Comment:
we now need `github.com/GoogleContainerTools/kpt/porch/api v0.0.0-20220404223850-4fe7ba260b65` which is already at HEAD so this file can be reverted.

Reply:
thanks.

Reply:
I'm sorry ... I was wrong. I thought my PR was already merged; alas, it wasn't. your code was correct.

Reply:
I pulled the latest and got v0.0.0-20220411164219-e3555a1d90a9. Shall I use this instead?